src/components/PurchaseOrder/PO.js

```diff
@@ -132,6 +152,8 @@ class PO extends Component {
       <PaneMenu>
         <IfPermission perm="purchase_order.item.put">
           <CloseOrderModal
+            closeOrderSubmit={this.closeOrder}
+            closingReasons={parentResources.closingReasons.records}
```

Comment:
I don't think you need to use `parentResources` here. It could be done through `resources`